src/ResourceManager/KeyVault/Commands.KeyVault/Commands.KeyVault.csproj

```diff
@@ -253,6 +255,10 @@
       <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
       <Name>Commands.ResourceManager.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\Profile\Commands.Profile\Commands.Profile.csproj">
```

Comment:
Please remove the reference to Commands.Profile, this is discouraged in cmdlet projects